cypress/e2e/layers.cy.ts

```diff
@@ -729,7 +769,10 @@ describe("layers", () => {
         type: "background",
       });
 
-      when.dragAndDropWithWait("layer-list-item:" + firstId, "layer-list-item:" + thirdId);
+      when.dragAndDropWithWait(
+        `layer-list-item:${firstId}`,
```

Comment:
I can't say these extra lines are really helpful when reading this code... But I guess it is what it is...